test/src/tests/generated/additional-properties/pets.go

```diff
@@ -31,6 +32,14 @@ func NewPetsClientWithBaseURI(baseURI string) PetsClient {
 
 // CreateAPInProperties create a Pet which contains more properties than what is defined.
 func (client PetsClient) CreateAPInProperties(ctx context.Context, createParameters PetAPInProperties) (result PetAPInProperties, err error) {
+	ctx = tracing.StartSpan(ctx, "generated/additional-properties/PetsClient.CreateAPInProperties")
```

Comment:
What does the generated string look like for the SDK?

Reply:
its going to be the fully qualified name

Reply:
btw not the most happy on how we determine that. pretty much we take the outputpath and do an index on it

Reply:
I see so something like `services/compute/mgmt/2016-03-30/compute/VirtualMachinesClient.CreateOrUpdate` then?

Reply:
yes, here is a sample, https://gist.github.com/vladbarosan/3ab6cd696e9b7413194ab78ed75225c3 . Was wondering actually, I could prefix with github.com/Azure/Azure-sdk-for-go

Reply:
Good idea.  It might be best to emit the package import path as an unexported const then concatenate that with the fully qualified method name.